bentoml/_internal/bento/build_dev_bentoml_whl.py

```diff
@@ -28,9 +28,8 @@ def build_bentoml_editable_wheel(target_path: str) -> None:
         return
 
     try:
-        from build.env import IsolatedEnvBuilder
-
         from build import ProjectBuilder
```

Comment:
I think there is some isort formatter bug for this specific file. I couldn't figure out why this is happening.